Makefile.js

```diff
@@ -650,7 +651,7 @@ target.gensite = function(prereleaseVersion) {
                 }
                 const added = versions.added[baseName];
 
-                if (!versions.removed[baseName] && !test("-f", sourcePath)) {
+                if (!versions.removed[baseName] && !fs.existsSync(sourcePath)) {
```

Comment:
Slightly different logic here, right? Before it was checking if it exists and if it's a file, not it just checks if it exists.